documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -304,6 +304,10 @@ spec:
     # ...
 ----
 
+*Strimzi Drain Cleaner image*
```

Comment:
Not sure we need this here.

Reply:
I would leave this out => it is something completely unrelated to the API.